CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v4.2.1 (Not Published Yet)
```

Comment:
Since this is a new feature, I think this should be a minor release 4.3.0, but we can update that when we're closer to publishing